gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java

```diff
@@ -86,8 +95,13 @@ public Response updateEntity(
                     ProxyBackendConfiguration backend =
                             OBJECT_MAPPER.readValue(jsonPayload, ProxyBackendConfiguration.class);
                     gatewayBackendManager.updateBackend(backend);
-                    log.info("Setting up the backend %s with healthy state", backend.getName());
-                    routingManager.updateBackEndHealth(backend.getName(), backend.isActive());
+                    log.info("Turning cluster %s %s", backend.getName(), backend.isActive() ? "on" : "off");
+                    routingManager.updateBackEndHealth(backend.getName(), backend.isActive() ? TrinoHealthStateType.PENDING : TrinoHealthStateType.UNHEALTHY);
+                    backendStateManager.updateStates(
+                            backend.getName(),
+                            ClusterStats.builder(backend.getName())
+                                    .healthy(backend.isActive() ? TrinoHealthStateType.PENDING : TrinoHealthStateType.UNHEALTHY)
```

Comment:
Extract a variable for repeated `backend.isActive() ? TrinoHealthStateType.PENDING : TrinoHealthStateType.UNHEALTHY`.